verifier/src/x509.rs

```diff
@@ -1,18 +1,22 @@
 // Copyright (c) 2023 The MobileCoin Foundation
 
 extern crate alloc;
-use alloc::string::{String, ToString};
+use alloc::string::String;
```

Comment:
Nit that's not directly related to this PR but I'm surprised imports are not being grouped here, I thought we standardized on grouping them (which might require a tweak to `rustfmt.toml`).

Reply:
It's not stable :(, https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#imports_granularity
the mobilecoin repo uses nightly while we had kept this and the SGX repo on stable, with a ci build for nightly.

Reply:
I fixed all of them, for now...
We may need to revisit if we want to start using nightly for these repos.

Reply:
Ha, shame on them. Seems like such a simple thing to stabilize 🤷‍♂️